src/blocks/Questions/Questions.tsx

```diff
@@ -22,6 +24,65 @@ const FaqMicrodataValues = {
     AnswerTextProp: 'text',
 } as const;
 
+const QuestionBlockItem = ({
```

Comment:
Can we create sub-directory for QuestionBlockItem?

Reply:
of course. done)